.ci/diff-report.sh

```diff
@@ -40,7 +40,7 @@ download-files)
   echo "Downloading files..."
 
   # check for projects link from PR, if not found use default from contribution repo
-  LINK="${LINK_FROM_PR:-$DEFAULT_PROJECTS_LINK}"
+  LINK="${DIFF_PROJECTS_LINK:-$DEFAULT_PROJECTS_LINK}"
```

Comment:
you need to change action code too, LINK_FROM_PR should be completely replaced with new value DIFF_PROJECTS_LINK

https://github.com/checkstyle/checkstyle/pull/15235#discussion_r1676976750